README.md

```diff
@@ -79,6 +79,15 @@ from django.core.urlresolvers import reverse_lazy
 LOGOUT_URL = reverse_lazy('loginas-logout')
 ```
 
+Additionally you can also specify the redirect url for logout default is `settings.LOGIN_REDIRECT_URL`
```

Comment:
Updated in commit [f0cb3f6
](https://github.com/skorokithakis/django-loginas/pull/49/commits/f0cb3f6261063f36bf791a1ff1b42fbf7757fe36) Thanks!